customer-commons/src/test/java/no/unit/nva/customer/model/CustomerDtoTest.java

```diff
@@ -27,6 +27,29 @@
 
 class CustomerDtoTest {
 
+    @Test
+    void doiAgentMergeTest() {
+        var doiAgent = randomCustomer()
+                           .getDoiAgent()
+                           .addPassword(randomString());
+
+        var secretDoi = new SecretManagerDoiAgentDao(randomCustomer().getDoiAgent());
```

Comment:
you can combine two random customers? They dont need to match?

Reply:
I am testing Merging two entities 
`  secretDoi.merge(doiAgent);`

Reply:
![image](https://user-images.githubusercontent.com/3939393/210978373-7703f038-df61-42ac-94d2-a18cd31e41b0.png)

Reply:
Merging of two objects.
Under normal circumstances (runtime), it will be impossible to have to different Id's in these objects.
CustomerId from request will be matched with CustomerId in Secrets, i.e. the secret you are updating should never change ID...